Library/Homebrew/download_strategy.rb

```diff
@@ -395,7 +395,7 @@ def initialize(url, name, version, **meta)
   def fetch(timeout: nil)
     end_time = Time.now + timeout if timeout
 
-    download_lock = LockFile.new(temporary_path.basename)
+    download_lock = DownloadLock.new(temporary_path.basename)
```

Comment:
You removed `.basename` from cleanup.rb but not this one. Just checking if intentional or not.